src/h3lib/lib/linkedGeo.c

```diff
@@ -290,20 +290,20 @@ static const LinkedGeoPolygon *findPolygonForHole(
  * @param root Root polygon including all loops
  * @return     0 on success, or an error code > 0 for invalid input
  */
-int normalizeMultiPolygon(LinkedGeoPolygon *root) {
+H3Error normalizeMultiPolygon(LinkedGeoPolygon *root) {
     // We assume that the input is a single polygon with loops;
     // if it has multiple polygons, don't touch it
     if (root->next) {
-        return NORMALIZATION_ERR_MULTIPLE_POLYGONS;
```

Comment:
Should we make new `H3Error` codes here? I'm not sure if it matters for the end user at this point, but this change drops information about the source of the error.

Reply:
We could - that information was not exposed to the user before.